dcicutils/deployment_utils.py

```diff
@@ -39,12 +39,13 @@ class Deployer:
 
     TEMPLATE_DIR = None
     INI_FILE_NAME = "production.ini"
+    INDEXER_ENTRY = 'ENCODED.INDEXER = "true"\n'
```

Comment:
I think the entries are all lowercase. I think the `.ini` file entry should be `encoded.indexer` or even just `indexer` given that the namespace is already encoded.